### PR TITLE
Basic Environment scene

### PR DESCRIPTION
**Pull request by lyhvictoria**
**Description**
Added in a basic environment scene for testing.  Made a Room prefab if the room is required in other scenes.
Currently, due to enclosed nature of the room, area light (which needs to be baked) is the best way to light up the room from the inside, resulting in some problems if duplicating the scene for individual feature testing is necessary. See below for explanation.

InsideView

**Scene Hierarchy**
image

@khooroko
@lyhvictoria

**Impact**
If impact is minor, no need for the reviewer to check out the branch to test locally before approval
- [x] Major
- [ ] Medium
- [ ] Minor

**Risks**
Any foreseeable difficulty in extending from this implementation?

There is currently no easy way to save light-maps for prefabs. Currently the easiest way to spawn prefabs with baked light is to have a separate scene pre-baked and to instantiate the prefabs in your scene from there. Since the only "prefab" that requires baked light for now is the room, I don't think we need to do that? But then there is no easy way to duplicate the environment scene without having to bake light again.

**Test Plan**
How to verify that this PR works and does not break anything else?

### CODE REVIEW

**Comment by khooroko** [comment]
The floor has too thin a collider, causing a high chance for things to clip through. Please remove the mesh collider from the floor and add a box collider that is thicker.
Also, merge the current master into this branch or rebase this branch atop current master. It's pretty outdated. Would be good to ensure that the CameraRig in your scene follows the one in the prefab.

Also remove MainCamera from your scene and consider adding BallManager so it's easy for us to see how balls work in the environment

**Comment by lyhvictoria** [comment]
@khooroko updated, pls review again

**Review by khooroko — APPROVED** [review]
👍 looks good

File: .gitignore [hunk 1]
```diff
@@ -16,7 +16,7 @@
 /[Aa]ssets/AssetStoreTools*
 
 # Visual Studio cache directory
-.vs/
```
Comment (khooroko) [inline .gitignore:19]: wait no add this line back, the lack of this is giving problems on my end

**Pull request merged**